pyvista/plotting/plotting.py

```diff
@@ -297,7 +297,8 @@ def untrack_mouse_position(self):
             del self._mouse_observer
 
 
-    def track_click_position(self, side="right", callback=None):
+    def track_click_position(self, side="right", callback=None,
```

Comment:
>  would recommend taking side.lower() and comparing it to a list that way the list needs to be less abondant for example here: "Right" will actually assign the left key ....

Ah thats a better way to handle this! will update...

> Also I reccommend an args check because here if the user rights 'foo' then it will be the left button that is clicked, maybe this is desirbale behaviour though ...

This is a good point, I'll check the `side` arg